templates/server-configmap.yaml

```diff
@@ -1,36 +1,37 @@
 {{- if $.Values.server.enabled }}
+{{- range $service := (list "frontend" "history" "matching" "worker") }}
```

Comment:
is the intention here is to create a different config map with a different `config_template.yaml` for every service?

Reply:
not a different tag but a different tag value?

but that tag value also maps to the env var that directs the service of which service profile to run.

this feels like something that should also be driven from that value.

https://github.com/temporalio/helm-charts/blob/4c7ee04165dc2bfe6dcbb67abbda6c0ac98521e7/templates/server-deployment.yaml#L98